website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
It's probably a bad place to start this convo here now that I'm thinking about it, I'm starting a thread in contributors mailing list instead so we can hash this out.

I am still blocking over the `(c)`, as it is not a legal assertion of copyright. Whether the author is ambiguous is TBD and non-blocking.